### PR TITLE
Adds step_threshold configuration to identify lengthy steps

### PR DESCRIPTION
**Pull request by asgrim**
Adds new optional `step_threshold` configuration option. Defaults to `0` which means reporting of individual steps is disabled (so `step_threshold` has to be manually specified to be used). The information is still actually collected, but just not output.

Fixes #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,16 @@ To configure the number of scenarios reported (default 10) specify the configura
...
+      report_length: 2
+```
+
+Optionally you may also set a threshold for individual steps to be reported. When the step threshold is `0`, the step threshold is ignored. The step threshold defaults to `0` (ignored):
+
+```yaml
+default:
+  extensions:
+    Brunty\Behat\SpeedtrapExtension:
+      step_threshold: 100 # this is in ms
```
Comment (Brunty) [inline README.md:47]: Wondering if we should also change the existing scenario threshold to be named `scenario_threshold` for consistency with the new threshold for steps.

_the joys of new projects and 0.X releases for BC breaks, eh?_ ;)

File: src/Listener/SpeedtrapListener.php [hunk 2]
```diff
@@ -18,18 +22,25 @@ class SpeedtrapListener implements EventSubscriberInterface
...
      */
-    public function __construct(Config $config, SpeedtrapLogger $speedtrapLogger)
+    private $stepLogger;
+
+    /**
+     * @param Config $config
+     * @param ScenarioLogger $speedtrapLogger
+     * @param StepLogger $stepLogger
+     */
+    public function __construct(Config $config, ScenarioLogger $speedtrapLogger, StepLogger $stepLogger)
```
Comment (Brunty) [inline src/Listener/SpeedtrapListener.php]: Rename property and parameter to `scenarioLogger` for consistency with new `stepLogger`

**Review by Brunty — CHANGES_REQUESTED** [review]

**Comment by Brunty** [comment]
Cheers for the work on this! <3

File: README.md (hunk 1 above)
Comment (asgrim) [reply README.md:47]: Renamed in 3fd5a39

File: src/Listener/SpeedtrapListener.php (hunk 2 above)
Comment (asgrim) [reply src/Listener/SpeedtrapListener.php]: Renamed in 3a6e5fa

**Review by Brunty — APPROVED** [review]

**Comment by Brunty** [comment]
❤️

**Pull request merged**